h/static/scripts/filter/document-domain.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var escape = require('escape-html');
```

Comment:
Should probably be called something else (`escapeHtml`?) as JS has a builtin called `escape` (which doesn't do what you want).